Source/WebCore/platform/cocoa/ParentalControlsURLFilter.mm

```diff
@@ -82,12 +81,26 @@ static bool wcrBrowserEngineClientEnabled()
     return [PAL::getWCRBrowserEngineClientClassSingleton() shouldEvaluateURLs];
 }
 
-ParentalControlsURLFilter& ParentalControlsURLFilter::singleton()
+static RefPtr<ParentalControlsURLFilter>& globalFilter()
 {
-    static MainThreadNeverDestroyed<UniqueRef<ParentalControlsURLFilter>> filter = UniqueRef(*new ParentalControlsURLFilter);
+    static NeverDestroyed<RefPtr<ParentalControlsURLFilter>> filter;
```

Comment:
Why did you get rid of the `MainThreadNeverDestroyed` ? Is this now accessed / modified from multiple threads? If so, it seems like we would need a Lock?